### ISSUE TITLE
Variable names conflict

### ISSUE THREAD

**Issue by 3bl3gamer** [benbria/coffee-coverage#38]
Simple test file:
```coffeescript
for a in [1,2]
	i = 100
	console.log(a)
```

After running `coffeeCoverage test.coffee test.js` js-file is generated:
```javascript
/* ... */
(function() {
  var a, i, len, ref;

  _$jscoverage["test.coffee"][1]++;

  ref = [1, 2];
  for (i = 0, len = ref.length; i < len; i++) {
    a = ref[i];
    _$jscoverage["test.coffee"][2]++;
    i = 100;
    _$jscoverage["test.coffee"][3]++;
    console.log(a);
  }

}).call(this);
```
And there is a bug: `i` variable is used in loop, although it is in original coffee source. Loop will exit after first iteration in this case.

After converting by `coffee -c test.coffee` it seems ok:
```javascript
// Generated by CoffeeScript 1.9.1
(function() {
  var a, i, j, len, ref;

  ref = [1, 2];
  for (j = 0, len = ref.length; j < len; j++) {
    a = ref[j];
    i = 100;
    console.log(a);
  }

}).call(this);
```

**Comment by jwalton** [comment]
Duplicate of #37.  I'll have this fixed shortly.

**Comment by jwalton** [comment]
Fixed in v0.4.5.
